tt_metal/impl/dispatch/device_command.hpp

```diff
@@ -215,7 +215,7 @@ class DeviceCommand {
 
     struct packet_ {
         CommandHeader header;
-        std::array<uint32_t, DeviceCommand::NUM_ENTRIES_IN_DEVICE_COMMAND - DeviceCommand::NUM_ENTRIES_IN_COMMAND_HEADER> data;
+        std::array<uint32_t, DeviceCommand::NUM_ENTRIES_IN_DEVICE_COMMAND - DeviceCommand::NUM_ENTRIES_IN_COMMAND_HEADER> data{};
```

Comment:
not sure why this is needed, this is on a perf path so I don't think we want to zero init if not necessary.  also this code is going away before long, replaced by the new fast dispatch command